application/actions/auth.go

```diff
@@ -30,6 +30,14 @@ const (
 	// http param for expires utc
 	ExpiresUTCParam = "expires-utc"
 
+	// http params for the Invite type and code
+	InviteCode        = "code"
+	InviteTypeMeeting = "meeting"
+
+	// session keys for using invites for authentication
+	InviteTypeSessionKey       = "InviteType"
+	InviteObjectUUIDSessionKey = "InviteObjectID"
+
```

Comment:
Maybe prefix the http param names with something like `QueryParam` and the sessions keys with `SessionKey`. (The session keys have it at the end, which seems a bit different than our norm.)

Actually, I just looked at the surrounding code and the "norm" for this file is to have the "category" at the end, so maybe just add `Param` to the first two, unless you feel like changing all of them.

Reply:
:+1: